Misanthropic-Mistletoes/reactive-nutshell
# Caroline four

# Description
Events should populate onto page in order sorted by date with the most recent event styled differently (bolds, larger text, and a background color). Events should delete. Events should be able to be edited. Made changes to Application Views by cleaning up and commenting out sections. 

Resolved merge conflicts. 


# Testing Instructions
1. Commit changes on your branch.
2. git checkout master
3. git fetch --all
4. git checkout caroline-four
5. run npm start
6. open VSC
7. check for changes listed in description above

# Checklist:
